doc/conf.py

```diff
@@ -12,14 +12,16 @@
 # All configuration values have a default; values that are commented out
 # serve to show the default.
 
+import os
 import sys
 import warnings
 import sphinx_gallery
 # import sphinx_bootstrap_theme
 from distutils.version import LooseVersion
 import matplotlib
 
-from doc.github_link import make_linkcode_resolve
+sys.path.insert(0, os.path.abspath("."))
```

Comment:
can you just move this line to below  l33 where there is a comment explaining what it does?

Reply:
(and also move the line below)

Reply:
Didn't take the time the read the comment. Thanks for pointing this out!